compiler/passes/scopeResolve.cpp

```diff
@@ -2797,6 +2797,7 @@ static bool symbolInBuiltinModule(Symbol* sym) {
   ModuleSymbol* mod = sym->getModule();
   if (mod->modTag == MOD_STANDARD &&
       (!strcmp(mod->name, "Builtins") ||
+       !strcmp(mod->name, "VectorizingIterator") ||
```

Comment:
I haven't looked at what `symbolInBuiltinModule` is used for, but it seems unfortunate to have to maintain this list in the compiler (and seems easy to forget to update if you're making a module builtin). Would this be better as a pragma on the module or something?

Separately maybe this list, however we determine it, could be used to squash chpldocs "Here's how to use this module" info?

Reply:
Yep. I think if we move it to a pragma on the module we can get chpldoc to squash the unnecessary part. I would like to make these changes in a separate PR (to merge after this one and the ones moving ChapelIO and ChapelEnv).

Reply:
BTW it is used for the functionality that lists available symbols.

Reply:
e.g. `trivial/diten/printvisible`

Reply:
PR #16861 will do it.